site/content/en/docs/tasks/_index.md

```diff
@@ -25,6 +25,7 @@ As a batch administrator, you can learn how to:
 - Setup [Sequential Admission with Ready Pods](/docs/tasks/setup_sequential_admission).
 - As a batch administrator, you can learn how to
   [monitor pending workloads](/docs/tasks/monitor_pending_workloads).
+- If you don't want to use pod's priority for workload, you can learn how to [run a Kueue managed Jobs with WorkloadPriority](/docs/tasks/run_job_with_workload_priority).
```

Comment:
```suggestion
-As a batch administrator, you can learn how to [run a Kueue managed Jobs with a custom WorkloadPriority](/docs/tasks/run_job_with_workload_priority).
```

This also might be more relevant for  batch users. And maybe administrators targeted section for the WorkloadPriority definition .